doc/scripting-api.md

```diff
@@ -0,0 +1,197 @@
+# Auditing scripts
+
+The auditor application supports running JavaScript files in order to perform the auditing procedure. The provided
+JavaScript runtime is compatible with [ECMAScript 2020 specification](https://262.ecma-international.org/11.0/) which is
+powered by GraalVM. For more info, check out
+[GraalVM JavaScript Compatibility](https://www.graalvm.org/reference-manual/js/JavaScriptCompatibility/).
+
+## Entry point and return value
+
+Each auditing script should define a function called `audit()` which takes no arguments. This function should do all the
+necessary steps in the auditing process and return an `AuditResult` at the end. The simplest possible auditing script is
+therefore:
+
+```javascript
+function audit() {
+    return AuditResult.of(true);
+}
+```
+
+## Script API
+
+Auditing scripts can call methods and read fields from the objects specified in this section. Apart from the API
+described here, several other global objects are available (e.g. `console` and `JSON`) which can also be used. IO
+operations, process and thread creation are not allowed in the auditing scripts (unless when available via API
+documented below). Auditing scripts can be tested via `POST /script/execute` when the application is running. This
+endpoint expects the script payload as `text/plain` content type.
+
+### Models
+
+Models are objects which are used primarily for the data transfer between the auditing script and its runtime.
+
+#### AuditResult
+
+Model of the result that should be returned from the `audit()` function.  
+Object name: `AuditResult`
+
+###### Fields
+
+No fields are readable.
+
+###### Methods
+
+| Signature | Description | Example call |
+| --------- | ----------- | ------------ |
+| `of(value: Boolean): AuditResult` | Constructs an `AuditResult` from the provided argument. | `AuditResult.of(true);` |
+
+#### List&lt;E&gt;
+
+Model of the lists returned from the API objects. Contains elements of type `E`.  
+No static object is provided in the scripts.
```

Comment:
Static objects are not provided in the script? There are no static objects in the script?